ethereum/migrations/2_next.js

```diff
@@ -148,13 +148,22 @@ module.exports = function (deployer, network, accounts) {
     // TODO: deploy the contract in this migration and use its address
     let administrator = accounts[0];
 
-    // Fee for incentivized channel
+    // Post-construction initialization for Basic outbound channel
+    if (!("BASIC_CHANNEL_PRINCIPAL" in process.env)) {
```

Comment:
is this set anywhere like in start-services script?

Reply:
ah, as per the PR description you need to set that in `ethereum/.env`. That's where other contract config goes (such as fee amounts for the incentivized channel).

Though in the near future (like next week) we need to make this configuration per chain, and ideally persisted somewhere other than env files.

Reply:
ah ok plz add to README and smth like .env.example

Reply:
Fixed in https://github.com/Snowfork/snowbridge/pull/458/commits/6105943822ad64ae0eab03bff4e2a33e12aea207